apps/mobile-wallet/src/screens/Addresses/Contact/ContactFormBaseScreen.tsx

```diff
@@ -50,9 +50,7 @@ const ContactForm = ({ initialValues, onSubmit, buttonText, headerOptions, ...pr
       contentPaddingTop
       headerOptions={{
         type: 'stack',
-        headerRight: () => (
-          <ContinueButton title={buttonText || t('Save')} onPress={handleSubmit(onSubmit)} iconProps={undefined} />
-        ),
+        headerRight: () => <ContinueButton title={buttonText || t('Save')} onPress={handleSubmit(onSubmit)} />,
```

Comment:
So as I've done it in other screens, the paradigm changed: main actions should be at the bottom of the screen, wrapped in a `BottomButtons` container.

Reply:
ok, then there are many screens that it this needs to be applied to.

Reply:
It can be done in a separate PR

Reply:
Applied to following screens:
- AddressFormBaseScreen
- ContactFormBaseScreen
- CustomNetworkScreen
- DecryptScannedMnemonicScreen